dbdump: changed location of DB dumps

ntarocco opened the pull request:
- mv db dumps to EOS
- change function to delete obsolete dumps

ntarocco commented:
Missing the tables to exclude, to be checked with @ludmilamarian, and test it.

ntarocco commented:
Deployed

ludmilamarian commented on modules/miscutil/lib/dbquery.py:
```diff
@@ -88,6 +91,9 @@
     CFG_DATABASE_SLAVE_SU_USER = ''
     CFG_DATABASE_SLAVE_SU_PASS = ''
     CFG_DATABASE_PASSWORD_FILE = ''
+    CFG_DBDUMPS_TEMP_DIR = ''
```
you should define these 3 in invenio.conf with default values.

ludmilamarian replied:
also the names should have started with `CFG_DATABASE_` in order for these to go directly to the `dbquery_config.py` file

ludmilamarian reviewed (commented):
Please check https://github.com/inveniosoftware/invenio/blob/master/CONTRIBUTING.rst#commit-messages for some style guidelines for commit messages. It's not a big deal, but it would be nice to keep the style for consistency.

ntarocco committed:
miscutil: change DB dumps location on disk

* Creates new db dumps in a local temporary folder on disk. Then,
  moves them to the configured EOS folder, expected to be mounted
  on the machine.

* Deletes previous db dumps from the EOS folder.

ntarocco commented:
@ludmilamarian thanks for your comments! I have fixed the issues, can you re-check it please?

ludmilamarian commented on modules/miscutil/lib/dbdump.py:
```diff
@@ -339,36 +330,108 @@ def _dbdump_run_task_core():
 
+def _new_kerberos_token():
+    """Renew Kerberos token."""
+    cmd = 'unset KRB5CCNAME && kinit -kt /opt/{0}.keytab {1} && eosfusebind'.format(
```
`/opt/{0}.keytab {0}` since you are using the same value.

ludmilamarian commented on config/invenio.conf:
```diff
@@ -107,7 +107,11 @@ CFG_DATABASE_SLAVE_SU_PASS =
+CFG_DATABASE_DBDUMP_TEMP_DIR = ''
+# Specify the user that have access to EOS to move db dumps to the EOS directory
+CFG_DATABASE_DBDUMP_EOS_USER = ''
+CFG_DATABASE_DBDUMP_EOS_DIR = ''
```
Don't forget to add them the `invenio-local.config` when deploying (and also to the `dbquery_config.py`)
Also, here in `invenio.conf` they should not have quotes around them (just empty instead of `"`)

ludmilamarian replied:
the commit message should be present tens. Also the module name should be `miscutils` instead of `dbdump`.

ntarocco commented:
@ludmilamarian hopefully I have applied all the modifications you have pointed out. Please feel free to add any other comment, it is important! ;)
